opensrp-reveal/src/test/java/org/smartregister/reveal/sync/RevealClientProcessorPowerMockTest.java

```diff
@@ -54,41 +67,156 @@ public void testProcessClientShouldCallProcessSprayEventWhenIsSprayEventType() t
         EventClient eventClient = new EventClient(event, null);
         eventClients.add(eventClient);
 
-        RevealClientProcessor clientProcessorSpy = spy(clientProcessor);
 
-        PowerMockito.doReturn(new ClientClassification()).when(clientProcessorSpy, "assetJsonToJava", anyString(), any());
-        PowerMockito.doReturn("").when(clientProcessorSpy, "processSprayEvent", any(Event.class), any(ClientClassification.class), anyBoolean());
-        clientProcessorSpy.processClient(eventClients, true);
+        PowerMockito.doReturn(new ClientClassification()).when(clientProcessor, "assetJsonToJava", anyString(), any());
+        PowerMockito.doReturn("").when(clientProcessor, "processSprayEvent", any(Event.class), any(ClientClassification.class), anyBoolean());
+        clientProcessor.processClient(eventClients, true);
 
-        PowerMockito.verifyPrivate(clientProcessorSpy, times(1)).invoke("processSprayEvent", eq(event), any(ClientClassification.class), anyBoolean());
+        PowerMockito.verifyPrivate(clientProcessor, times(1)).invoke("processSprayEvent", eq(event), any(ClientClassification.class), anyBoolean());
     }
 
     @Test
     public void testProcessClientShouldCallProcessMosquitoCollectionEventWhenIsMosquitoCollectionEventType() throws Exception {
-        mockStatic(Utils.class);
-        mockStatic(PreferencesUtil.class);
-
-        PreferencesUtil preferencesUtil = mock(PreferencesUtil.class);
-        PowerMockito.when(preferencesUtil.getCurrentOperationalArea()).thenReturn("");
-
-        PowerMockito.when(PreferencesUtil.getInstance()).thenReturn(preferencesUtil);
-        PowerMockito.when(Utils.getOperationalAreaLocation(anyString())).thenReturn(new Location());
-
-        RevealClientProcessor clientProcessor = Whitebox.newInstance(RevealClientProcessor.class);
-
+        mockStaticMethods();
         List<EventClient> eventClients = new ArrayList<>();
         Event event = new Event();
         event.setEventType(MOSQUITO_COLLECTION_EVENT);
 
         EventClient eventClient = new EventClient(event, null);
         eventClients.add(eventClient);
 
-        RevealClientProcessor clientProcessorSpy = spy(clientProcessor);
+        PowerMockito.doReturn(new ClientClassification()).when(clientProcessor, "assetJsonToJava", anyString(), any());
+        PowerMockito.doReturn("").when(clientProcessor, "processMosquitoCollectionEvent", any(Event.class), any(ClientClassification.class), anyBoolean());
```

Comment:
Remove this, the object under test shouldn't be disabled from executing methods on itself.

Reply:
Done.